model/fieldtypes/Time.php

```diff
@@ -26,22 +34,25 @@ public function setValue($value, $record = null) {
 	}
 
 	/**
-	 * Return a user friendly format for time
-	 * in a 12 hour format.
-	 *
-	 * @return string Time in 12 hour format
+	 * Returns the time in the format specified by the config value niceFormat, or 12 hour format by default 
+	 * e.g. "3:15pm"
+	 * 
+	 * @return string
 	 */
 	public function Nice() {
-		if($this->value) return date('g:ia', strtotime($this->value));
+		if($this->value) return $this->Format(Config::inst()->get(__CLASS__, 'niceFormat'));
+		
 	}
 
 	/**
 	 * Return a user friendly format for time
 	 * in a 24 hour format.
-	 *
+	 * 
 	 * @return string Time in 24 hour format
+	 * @deprecated 3.2 Use Nice() with config setting niceFormat instead
 	 */
 	public function Nice24() {
+		Deprecation::notice('3.2', 'Use Nice() with config setting '.__CLASS__.'::niceFormat instead');
```

Comment:
I'd make this a separate config rather than deprecating it. Saying to use another method and changing the config defeats the purpose of having two methods; It's useless having it if you can only have one global format for your project. What if you want $Nice in one place and $Nice24 in another?